### PR TITLE
Ballistics - Fix DMR_03 (SIG556) initSpeed

### PR DESCRIPTION
**Pull request by Laid3acK**
**When merged this pull request will:**
- Fix the `DMR_03_base_F` `initSpeed`.
- Fix the missing `arifle_katiba_Base_F` 30 rnd IR-Dim magazine (KH2002 Sama variants).
- Fix the `ACE_30Rnd_556x45_Stanag_Tracer_Dim` picture (30 rnd magazine icon instead 20 rnd).

### CODE REVIEW

File: addons/ballistics/CfgWeapons.hpp [hunk 1]
```diff
@@ -51,12 +51,15 @@ class CfgWeapons {
...
         ACE_barrelLength = 508.0;
         ACE_barrelTwist = 254.0;
-        initSpeed = -0.991536;
         magazineWell[] += {
             "CBA_762x51_M14"
         }; // vanilla magazineWell[] = {"M14_762x51"};
     };
 
+    class srifle_DMR_03_F: DMR_03_base_F {
+        initSpeed = -0.991536;
```
Comment (commy2) [inline addons/ballistics/CfgWeapons.hpp]: Explain.

**Comment by commy2** [comment]
Why all the empty lines?

File: addons/ballistics/CfgWeapons.hpp (hunk 1 above)
Comment (Laid3acK) [reply addons/ballistics/CfgWeapons.hpp]: When i tested the RC, i saw that the `class srifle_DMR_03_F` has his own `initSpeed `, 940 if i remember correctly.

**Comment by Laid3acK** [comment]
I believed it was a part of the Coding Guidelines. I'm wrong ?

File: addons/ballistics/CfgWeapons.hpp (hunk 1 above)
Comment (commy2) [reply addons/ballistics/CfgWeapons.hpp]: Yeah, saw that. I think we should keep the value in both classes and put a comment after this line.

File: addons/ballistics/CfgWeapons.hpp [hunk 2]
```diff
@@ -51,12 +51,15 @@ class CfgWeapons {
...
         ACE_barrelLength = 508.0;
         ACE_barrelTwist = 254.0;
-        initSpeed = -0.991536;
         magazineWell[] += {
             "CBA_762x51_M14"
         }; // vanilla magazineWell[] = {"M14_762x51"};
     };
 
+    class srifle_DMR_03_F: DMR_03_base_F {
+        initSpeed = -0.991536; // Default vanilla value 940
```
Comment (commy2) [inline addons/ballistics/CfgWeapons.hpp]: No, an explanation why this is here. And also keep the original change.

Comment (Laid3acK) [reply addons/ballistics/CfgWeapons.hpp]: I don't understand what do you mean with an explanation. There is no explanation with the old config.
>And also keep the original change
Same, i don't understand my mistake.

**Comment by Laid3acK** [comment]
Reverted to the old format (better compatibility with third mods).

**Pull request merged**